.github/workflows/main.yml

```diff
@@ -10,31 +10,48 @@ on:
       - v5
 
 jobs:
+  # Note: changing the name of the job disables Slither checks with the error:
+  # Warning: Code scanning cannot determine the alerts introduced by this pull
+  # request, because 1 configuration present on refs/heads/develop was not found:
+  #
+  # Actions workflow (main.yml)
+  # ❓ .github/workflows/main.yml:coverage
   coverage:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Init
+      - name: Set up Nodejs
+        uses: actions/setup-node@v3
```

Comment:
v4 would be nice :) https://github.com/actions/setup-node?tab=readme-ov-file#usage

Reply:
Yes [Use setup node action v4](https://github.com/iExecBlockchainComputing/PoCo/pull/178/commits/75f0084528c0461e8354cbefc4550fd9382e7693)